CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
```

Comment:
Please remove this file completely, the Releases page is used instead 🙏 Or, change the version to 3.1.0. Or don´t touch this file in this PR, and I will remove this file in another PR.

Reply:
Okay, please remove the file in another PR I will undo my changes.